internal/pkg/manifest/validate_test.go

```diff
@@ -1541,105 +1541,35 @@ func TestRoutingRuleConfiguration_validate(t *testing.T) {
 		"return if routing rule configuration is nil": {},
 		"error if both target_container and targetContainer are specified": {
 			RoutingRuleConfiguration: RoutingRuleConfiguration{
-				MainRoutingRule: ALBRoutingRule{
+				Main: RoutingRule{
 					Path:            stringP("/"),
 					TargetContainer: aws.String("mockContainer"),
 				},
 				TargetContainerCamelCase: aws.String("mockContainer"),
 			},
 			wantedError: fmt.Errorf(`must specify one, not both, of "target_container" and "targetContainer"`),
 		},
-		"error if one of allowed_source_ips is not valid for additional routing rules": {
```

Comment:
I see these test cases were deleted because of https://github.com/aws/copilot-cli/pull/4622#discussion_r1137435060. Just wanted to make sure I didn't miss anything: were they moved to `func TestALBRoutingRule_validate(t *testing.T)` or were they simply deleted?

Reply:
Okay so test cases that I removed are already present in the `TestALBRoutingRule_validate`. I added similar test here just to make sure that `additional_rules` also gets tested which it does through `TestALBRoutingRule_validate`. But just to make sure we get proper error message with prefix `validate "additional_rules[0]"` I added similar tests with additional_rules here. Does that make sense? So just removing them is fine.